python/ray/dataframe.py

```diff
@@ -0,0 +1,467 @@
+import pandas as pd
+import numpy as np
+import ray
```

Comment:
I think it makes sense to move the file to `python/ray/dataframe/dataframe.py`. And to add a file `python/ray/dataframe/__init__.py` that has some lines like

```
from .dataframe import DataFrame
```